services/repository/src/main/java/org/sagebionetworks/repo/web/controller/MessageController.java

```diff
@@ -96,6 +96,16 @@ MessageToUser createMessage(
 		return serviceProvider.getMessageService().create(userId, toCreate);
 	}
 	
+	@ResponseStatus(HttpStatus.NO_CONTENT)
```

Comment:
Add javadoc that this is for use with CloudMailIn service, not clients in general.